iHerb_autotests/iherb_checkout_page.py

```diff
@@ -0,0 +1,39 @@
+from selene import browser, by, be, query
+
+from iherbtestvenv.iherb_payment_update_page import IherbPaymentUpdatePage
+from iherbtestvenv.iherb_thank_you_page import IherbThankYouPage
+
+class IherbCheckoutPage:
+
+    def get_product_name(self, product_number):
+        browser.element(by.xpath("//*[name()='svg' and @class='LandingPanel__ToggleIcon-sc-bh59r1-4 fUUZvX']"))\
+            .wait.for_(be.visible)
+        browser.element(by.xpath("//*[name()='svg' and @class='LandingPanel__ToggleIcon-sc-bh59r1-4 fUUZvX']"))\
```

Comment:
extract to variable, re-use

Reply:
Can you please explain your comment in more detail?
Not sure, that I understand it.